examples/walkthrough/README.md

```diff
@@ -94,7 +194,7 @@ However, often you want to have two different containers that work together.  An
 apiVersion: v1
 kind: Pod
 metadata:
-  name: www
+  name: git-monitor
```

Comment:
`www` is a terribly generic name. what are we serving? `/srv/www` doesn't exist anywhere...